### PR TITLE
Nbr peer version infos interface

### PR DESCRIPTION
**Pull request by likaioc**

### CODE REVIEW

File: p2pserver/actor/server/actor.go [hunk 1]
```diff
@@ -243,3 +246,26 @@ func (this *P2PActor) handleTransmitConsensusMsgReq(ctx actor.Context, req *Tran
...
+		return
+	}
+
+	nbrPeers := this.server.GetNetWork().GetNeighbors()
+	nbrResp := &GetNbrPeerVersionInfosRsp{
+		VersionInfos: []*common.NbrPeerVersionInfo{},
+	}
+
+	var p *peer.Peer
+	for _, p = range nbrPeers {
```
Comment (Honglei-Cong) [inline p2pserver/actor/server/actor.go]: for _, p := range nbrPeers

File: p2pserver/common/p2p_common.go [hunk 2]
```diff
@@ -132,6 +122,16 @@ const (
 	DISCONNECT_TYPE  = "disconnect" //peer disconnect info raise by link
 )
 
+//PeerAddr represent peer`s net information
+type PeerAddr struct {
```
Comment (Honglei-Cong) [inline p2pserver/common/p2p_common.go:126]: un-necessary change.

File: p2pserver/common/p2p_common.go [hunk 3]
```diff
@@ -152,6 +152,11 @@ type AppendBlock struct {
 	MerkleRoot com.Uint256  // MerkleRoot
 }
 
+type NbrPeerVersionInfo struct {
```
Comment (Honglei-Cong) [inline p2pserver/common/p2p_common.go]: move to p2pserver/actor/server/common.go ?

**Comment by Honglei-Cong** [comment]
please clean up the commit log

seems lots of unrelated commit logs,

File: p2pserver/actor/server/common.go [hunk 4]
```diff
@@ -132,3 +132,15 @@ type TransmitConsensusMsgReq struct {
 	Target uint64
 	Msg    ptypes.Message
 }
+
+type GetNbrPeerVersionInfosReq struct {
+}
+
+type NbrPeerVersionInfo struct {
+	ID      uint64
+	Version uint32
```
Comment (laizy) [inline p2pserver/actor/server/common.go]: I think what we need is the node's release version, not p2p protocal version.

Comment (likaioc) [reply p2pserver/actor/server/common.go]: Ok, have changed to node's release version.

**Comment by Honglei-Cong** [comment]
new pr: https://github.com/ontio/ontology/pull/761

**Pull request closed without merging**